hydrus/helpers.py

```diff
@@ -224,17 +224,14 @@ def finalize_response(path: str, obj: Dict[str, Any]) -> Dict[str, Any]:
     collections, parsed_classes = get_collections_and_parsed_classes()
     expanded_base_url = DocUrl.doc_url
     if path in collections:
-        # path is of a collection class
-        collection = get_doc().collections[path]["collection"]
-        class_name = collection.manages["object"].split(expanded_base_url)[1]
-        collection_manages_class = get_doc().parsed_classes[class_name]["class"]
-        collection_manages_class_type = collection_manages_class.title
-        collection_manages_class_path = collection_manages_class.path
         members = list()
-        for member_id in obj["members"]:
+        for member in obj["members"]:
+            member_id = member[0]
```

Comment:
This needs to be modified so that we can use the discussed format for the request.

Reply:
Yes. I am working on this right now